webhooks/github_feature_tracker.cgi

```diff
@@ -28,13 +28,9 @@ PASSWORD = '*******'
 
 # to get the right number for the milestones:
 # https://api.github.com/repos/qgis/QGIS-Documentation/milestones
-# QGIS 3.0 = 7
-# QGIS 3.2 = 9
-# QGIS 3.4 = 10
-# QGIS 3.6 = 11
-# QGIS 3.8 = 12
 # QGIS 3.10 = 13
 # QGIS 3.16 = 14
+# QGIS 3.22 = 15
```

Comment:
is this right: QGIS 3.22 ? Or should this be 3.18 ?

Reply:
3.22 the milestone is the next LTR

Reply:
But these milestone numbers, they are... coming from us or from github?
In the other part of you patch you translate milestone 15 to 3.18... And here we do 15 is 3.22 ?
I think these milestones are not 'our' numbers is it?

Reply:
> But these milestone numbers, they are... coming from us or from github?

https://api.github.com/repos/qgis/QGIS-Documentation/milestones
3.18 is a label, helping us to know in which of the three versions concerned by the doc the feature was added (eg to add "new in xxx" next to functions or algorithms)

Reply:
Looks like this https://github.com/qgis/QGIS-Documentation/issues?q=is%3Aissue+author%3AQGIS-feature-tracker+label%3A3.18

Reply:
But is that ok? I mean, are we postponing new features for 3.22 now? Instead of the following 3.18?
I'll ask on the mailing list

Reply:
I'm not sure I understand what you mean.
Docs are released only for LTR, right? Then after 3.16 LTR (which _shouldn't_ receive new features), the next one will be 3.22 LTR. So new issue reports in doc repo are 3.22 docs targets. There will be no 3.18 docs. But the 3.22 docs (which is our current master) will deliver features added from QGIS 3.18 to 3.22.
Am I missing something?

Reply:
Never mind, I'm not fully following. But is probably all right.

Reply:
😄 I think it's right and I hope I'm not the only one understanding the system...
The confusing part is that milestones and labels both refer to qgis release numbers: the former is the version of the docs (3.10, 3.16, 3.22... LTR) and the latter represents the application's version in which the feature is added. Let's discuss this later in f2f and maybe could it be improved.